airflow/configuration.py

```diff
@@ -551,18 +508,20 @@ def _get_env_var_option(self, section, key):
 
     def _get_cmd_option(self, section, key):
```

Comment:
There was a PR a while ago to allow certain configuration options to be specified as bash commands. That way passwords could be kept even more securely hidden, and retrieved via command.